src/components/pages/profile/AddProfileForm.js

```diff
@@ -3,6 +3,7 @@ import ProfileInputs from "./ProfileInputs";
 
 function AddProfileForm({ handleSubmit, addHandleChange }) {
   return (
+  
```

Comment:
For some reason the first name, last name, and state are not saving, so they don't appear in the profile info. Take a look please at how the inputs work and make sure it saves that. It could be just some misspelling error somewhere.

Reply:
first/last name now working State is is in debug.